domains/kryptx.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "0kryptx",
+        "email": "k3px@proton.me"
+    },
+    "record": {
+        "CNAME": "kryptx.github.io"
```

I cannot reach this website, did you mean `0kryptx.github.io`?
```suggestion
        "CNAME": "0kryptx.github.io"
```

---

oh yeah !